source/common/upstream/upstream_impl.cc

```diff
@@ -865,7 +868,7 @@ void StrictDnsClusterImpl::ResolveTarget::startResolve() {
 
         HostVector hosts_added;
         HostVector hosts_removed;
-        if (parent_.updateDynamicHostList(new_hosts, hosts_, hosts_added, hosts_removed, false)) {
+        if (parent_.updateDynamicHostList(new_hosts, hosts_, hosts_added, hosts_removed)) {
```

Comment:
Can you look and see if it would be a small amount of effort to add a test for static DNS along the same lines as EDS? I think it would be good to have coverage in both cases. This implications of this code are scary and the more coverage we have the better IMO.

Reply:
wasn't too bad, added